core/editor/loader/editor-loader-factory.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Elementor\Core\Editor\Loader;
+
+use Elementor\Core\Editor\Editor;
+use Elementor\Core\Editor\Loader\V1\Editor_V1_Loader;
+use Elementor\Core\Editor\Loader\V2\Editor_V2_Loader;
+use Elementor\Core\Utils\Collection;
+use Elementor\Plugin;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Editor_Loader_Factory {
+	/**
+	 * @return Editor_Loader_Interface
+	 */
+	public static function create() {
+		$is_editor_v2_active = Plugin::$instance->experiments->is_feature_active( Editor::EDITOR_V2_EXPERIMENT_NAME );
+
+		// Nonce verification is not required, using param for routing purposes.
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$editor_version = Utils::get_super_global_value( $_GET, 'v' ) ?? ( $is_editor_v2_active ? '2' : '1' );
+
+		$config = new Collection( [
+			'assets-url' => ELEMENTOR_ASSETS_URL,
+			'assets-path' => ELEMENTOR_ASSETS_PATH,
+			'min-suffix' => ( Utils::is_script_debug() || Utils::is_elementor_tests() ) ? '' : '.min',
```

Comment:
Not sure that I understand what is the advantage of the config being outside of the loader.. isn't it always the same?

Reply:
Feels like this configuration comes from the outside and the loader itself should not concerned about the logic of this configuration.
But if you think it will be cleaner to initiate it inside the loader it is fine by me